mosquitto/DOCS.md

```diff
@@ -29,6 +29,8 @@ If you have old MQTT settings available, remove this old integration and restart
 
 ## Configuration
 
+After the initial installation process described above, the configuration editor can be found in **Supervisor** -> **Mosquitto broker** -> **Configuration** tab.
```

Comment:
From my point of view, this applies to all add-ons. With the recent UI changes it could make sense to add it somewhere at https://www.home-assistant.io/hassio/

Reply:
Ah OK, yes that's fair.
I think what caught me out here is I arrived at the Add-on instructions having already read through the instructions on https://www.home-assistant.io/docs/mqtt/broker/ (being the top Google hit for "home assistant mosquitto config") and that page describes editing `configuration.yaml` so I was still in that mindset when eventually finding these Add-on docs (and being new to Add-ons didn't know this wider convention)
Thanks